packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -84,27 +63,7 @@ $bpk-spacing-v2: true;
   }
 
   &__item-expand-icon {
-    @include bpk-themeable-property(
-      fill,
-      --bpk-accordion-color,
-      $bpk-color-sky-blue-shade-03
-    );
-
-    @include bpk-hover {
-      @include bpk-themeable-property(
-        fill,
-        --bpk-accordion-hover-color,
-        $bpk-color-sky-blue-shade-03
-      );
-    }
-
-    &:active {
-      @include bpk-themeable-property(
-        fill,
-        --bpk-accordion-active-color,
-        $bpk-color-sky-blue-shade-03
```

Comment:
do we need to also remove this vars where they were declared? (maybe not, just to know)

Reply:
same 😬